gcloud/credentials.py

```diff
@@ -207,7 +218,11 @@ def _get_signature_bytes(credentials, string_to_sign):
         pkey = _get_pem_key(credentials)
         if not isinstance(string_to_sign, six.binary_type):
             string_to_sign = string_to_sign.encode('utf-8')
-        return crypto.sign(pkey, string_to_sign, 'SHA256')
+        if crypto is None:
```

Comment:
Yes I considered that this error would never get a chance to occur in "real life" but put it in anyhow.

These just make me want to upstream this code [into `oauth2client`][1] even more.

[1]: https://github.com/google/oauth2client/blob/b13ab40a01adacf0075fd52f95173abb33297254/oauth2client/service_account.py#L265-L266